content/basics.article

```diff
@@ -1,131 +1,131 @@
-Packages, variables, and functions.
-Learn the basic components of any Go program.
+Pakiety, zmienne, oraz funkcje.
+Naucz się podstawowych komponentów każdego programu Go.
 
-The Go Authors
+Autorzy Go
 https://golang.org
 
-* Packages
+* Pakiety (Packages)
 
-Every Go program is made up of packages.
+Każdy program Go jest stworzony z pakietów.
```

Comment:
W calym tekscie jest bardzo duzo tlumaczen ktorych bym nie robil.

`packages` jest uzywane w kodzie. Tlumaczac ta nazwe utrudniamy zrozumienie. 

Jak to zrobic:

```
Kazdy program posiada jeden lub wiecej pakietow tzw. packages.
packages zwykle uzywane sa na poczatku pliku
```

@MStokluska

Reply:
Bardzo dobry pomysl. Wrzuce poprawki.
Dzieki za review!